go/vt/vttablet/tabletmanager/vreplication/framework_test.go

```diff
@@ -141,7 +141,6 @@ func setup(ctx context.Context) (func(), int) {
 	resetBinlogClient()
 
 	vttablet.InitVReplicationConfigDefaults()
-	vttablet.DefaultVReplicationConfig.ExperimentalFlags = 0
```

Comment:
This is where the flags were disabled for all of the unit tests and I did not notice it previously.

Reply:
If we want this PR to be able to stand on its own, I could change this line to `vttablet.DefaultVReplicationConfig.ExperimentalFlags = 7` rather than removing it.